test/solidlsp/java/test_java_basic.py

```diff
@@ -1,13 +1,17 @@
 import os
+import platform
 
 import pytest
 
 from solidlsp import SolidLanguageServer
 from solidlsp.ls_config import Language
 from solidlsp.ls_utils import SymbolUtils
 
+# These marks will be applied to all tests in this module
+# We skip the tests if not on Windows (CI hangs on Ubuntu for unknown reasons; testing on Windows is sufficient)
+pytestmark = [pytest.mark.java, pytest.mark.skipif(platform.system() != "Windows", reason="Java tests are only run on Windows")]
```

Comment:
It's better to do the exclusion only in CI, the tests do run through on "normal" linux environments. We already have a platform dependent if-else there

Reply:
Updated.